srcpkgs/x2goserver/template

```diff
@@ -1,28 +1,53 @@
 # Template file for 'x2goserver'
 pkgname=x2goserver
 version=4.1.0.3
-revision=1
+revision=2
 build_style=gnu-makefile
-hostmakedepends="pkg-config perl"
+hostmakedepends="pkg-config perl which"
 makedepends="libssh2-devel"
-depends="perl perl-Config-Simple perl-DBI perl-Capture-Tiny perl-DBD-SQLite bash iproute2 makepasswd openssh lsof xauth perl-File-BaseDir nx-libs perl-File-Which"
+depends="bash iproute2 openssh lsof xauth xhost xwininfo pwgen psmisc xkeyboard-config
+ perl perl-Config-Simple perl-DBI perl-Capture-Tiny perl-DBD-SQLite perl-File-BaseDir
+ perl-File-Which perl-Try-Tiny perl-Switch nx-libs"
 short_desc="Open source graphical Remote Desktop based on NX technology"
-maintainer="eoli3n <jkirsz@gmail.com>"
+maintainer="eoli3n <jonathan.kirszling@runbox.com>"
 license="GPL-2.0-or-later"
 homepage="http://www.x2go.org"
 distfiles="http://code.x2go.org/releases/source/${pkgname}/${pkgname}-${version}.tar.gz"
 checksum=6776aaa354f5a44e349f0b3c176d4988c88a618c2edf46c98a37ae89c069dcd0
 system_groups="x2gouser"
-system_accounts="x2gouser"
 x2gouser_homedir="/var/lib/x2go"
 x2gouser_shell="/bin/false"
 
-post_extract() {
-	# Replace hard-coded /usr/sbin
-	for _file in x2goserver-printing/Makefile x2goserver/Makefile x2goserver-xsession/Makefile libx2go-server-db-perl/Makefile x2goserver-common/Makefile; do
-		vsed -e 's,/sbin,/bin,g' -i "$_file"
+post_patch() {
+    # Replace hard-coded /usr/local
+	for _file in \
+	  	Makefile \
+	  	x2goserver-desktopsharing/Makefile \
+	  	x2goserver-desktopsharing/bin/x2goterminate-desktopsharing \
+	  	libx2go-server-db-perl/Makefile \
+	  	x2goserver-fmbindings/Makefile \
+	  	x2goserver-xsession/Makefile \
+	  	x2goserver-x2goagent/Makefile \
+	  	x2goserver-common/Makefile \
+	  	x2goserver/Makefile \
+	  	x2goserver-extensions/Makefile \
+	  	x2goserver-printing/Makefile \
+	  	x2goserver/man/man8/x2gobasepath.8 \
+	  	x2goserver/man/man8/x2gopath.8; do
```

Comment:
Only perl and shell scripts in destdir/usr/{bin,lib} with no references to /usr/local when i remove vseds

Reply:
Building/installing then test gives the first error reported
```
/usr/lib/x2go/x2gocheckport: ligne 131: '/usr/local/lib/x2go/libx2go-server-db-sqlite3-wrapper.pl'
```

Reply:
Adding ``make_build_args="PREFIX=/usr".`` doesn't solve it.

Reply:
Ok, try rebuilding with `make_build_args="PREFIX=/usr"` to see if that fixes the problem.

Reply:
I forgot to install my rebuilt package, i confirm that make_build_args fixes.

Reply:
I'm not in front of the computer, but you can probably do
```bash
sed -i -e '/PREFIX/s,/usr/local,/usr,' */Makefile
```
to avoid enumerating all of the makefiles.